xml/cap_depl_service_broker_sidecar.xml

```diff
@@ -283,22 +283,22 @@ suse/uaa                     2.13.3  A Helm chart for SUSE UAA</screen>
     xC8x/+zT0QkvcRJBio5gg670+25KJQ==
     -----END CERTIFICATE-----
    
-   # UAA certificate
+  # UAA certificate
    UAA_CA_CERT: |
```

Comment:
This line (along with the others in this section) should align with the comment line.